src/screens/Donate/Donate.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { Fragment } from 'react';
+import { Grid } from '@material-ui/core';
+import { Icon } from '@iconify/react';
+import testTube from '@iconify/icons-twemoji/test-tube';
+import loveLetter from '@iconify/icons-twemoji/love-letter';
+import deliveryTruck from '@iconify/icons-twemoji/delivery-truck';
+import {
+  EmbedWrapper,
+  ContentWrapper,
+  SectionHeader,
+  ContentSection,
+  ListHeader,
+  BodyCopy,
+  ListSection,
+} from 'screens/Donate/Donate.style';
+import PartnersSection from 'screens/HomePage/PartnersSection';
```

Comment:
nit: Consider moving `PartnersSection` to its own folder in `components` and import it from both `HomePage` and `Donate` screens.